Added current evaluated `querystring` as a parameter in expander `querystring` key when it is a function (#7012)
plone/volto: docs/source/configuration/settings-reference.md

>     The configuration accepts a list of matchers, with the ability to filter by the request path and action type for maximum flexibility.
>     It also accepts a `querystring` object that allows configuring the expanders via query string parameters, such as the navigation expander.
>     The `querystring` object accepts a querystring object or a function that returns a querystring object.

Reviewer: ```suggestion
    `config` accepts a list of matchers to filter by request path and action type for maximum flexibility.
    It also accepts a `querystring` object that allows to configure the expanders via querystring parameters, for example, the navigation expander.
    The `querystring` object accepts a querystring object or a function that returns a querystring object.
```
